src/index.ts

```diff
@@ -38,6 +39,8 @@ export function createApi<
   endpoints,
   keepUnusedDataFor = 60,
```

Comment:
we could also add this to the config state

Reply:
Added